packages/mrml-python/tests/test_thread_safety.py

```diff
@@ -0,0 +1,163 @@
+import concurrent.futures
```

Comment:
Maybe this isn't totally needed since Rust itself is generally thread safe, but it runs fast.